cypress/e2e/Feedback.feature

```diff
@@ -19,7 +19,16 @@ Feature: Feedback
             | Select a rating for this service |
             | Do not forget to leave your feedback in the box |
         And I see "Error" in the title
-        When I select satisfied
+        # click through all the radio buttons due to history of bug in this area (see LPAL-1038)
```

Comment:
We probably need to do something more than this to prove that the radio button is really being selected. (Clicks on the radio buttons do work, it's just that the value isn't set correctly.) Could be as simple as checking that input on the form; or could intercept the submitted form and check the value of the radio there. The latter would be more complicated but probably more indicative that the form is working properly.

Reply:
Done

Reply:
I was expecting some extra assertions that the `rating` input on the form has the expected value. Am I missing something?

Reply:
Whoops. Had committed on wrong git branch. 🤦  Should be sorted now